wcfsetup/install/files/lib/system/template/plugin/PagesFunctionTemplatePlugin.class.php

```diff
@@ -160,6 +64,10 @@ public function execute($tagArgs, TemplateEngine $tplObj)
                 $parameters['application'] = $tagArgs['application'];
             }
 
+            // The previous implementation required the `pageNo=%d` placeholder
+            // to be present in the link argument.
+            $tagArgs['link'] = \preg_replace('~(^pageNo=%d&?|&pageNo=%d)~', '', $tagArgs['link']);
```

Comment:
```suggestion
            $tagArgs['link'] = \preg_replace('~(?:^pageNo=%d(?:&|$))|(?:&pageNo=%d(?=&|$))~', '', $tagArgs['link']);
```

This regular expression is a little more defensive.